vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/catalogbundleinitialization.rb

```diff
@@ -16,12 +16,28 @@
 #
 # Inputs: $evm.root['service_template_provision_task'].dialog_options
 #
+# array_entry example - Classification::1
+def vmdb_object_from_array_entry(entry)
+  model, id = entry.split("::")
+  $evm.vmdb(model, id.to_i) if model && id
+end
+
+def array_value(hash, hash_key, value)
+  if hash.key?(hash_key)
+    values = hash[hash_key].split(',')
+    values << value
+    values.map { |str| "#{str}" }.join(',')
```

Comment:
You don't need the block+interpolation...it can just be `values.collect(&:to_s).join(",")`